assets/header_template.go

```diff
@@ -0,0 +1,24 @@
+package assets
```

Comment:
I suggest to not put static template in the `*.go` file. Each modification will require a full build.

Please check the implementation in configs/config.go. It reads from the `$EYWA_HOME/configs/*.yml`

So you can create a director with name `templates` as you like. And put the static template there. 

For `c` headers, you can do `templates/header.c`, each rendering then can read from the file. 

This way, it's both easy to read and debug. Also encourages people to contribute more such client files.